tests/test.spec.ts

```diff
@@ -1,18 +1,78 @@
 import { join } from 'path';
 import * as rimraf from 'rimraf';
 import * as shell from 'shelljs';
+import { newImportStatementRegex, newStringRegex } from '../dist/utils';
 
 const projectsRoot = join(__dirname, '../projects');
 
-[1, 3, 4, 5, 6, 7, 8].forEach((value) => {
-  it(`Project ${value}`, () => {
-    const projectDir = join(projectsRoot, `project${value}`);
-    rimraf.sync(join(projectDir, 'dist'));
-    const { code, stderr } = shell.exec('npm start', {
-      cwd: projectDir,
-      silent: true
-    });
-    if (code !== 0) console.error(stderr);
-    expect(code).toEqual(0);
+/**
+ * Sample JavaScript code with a bunch of require/import
+ * statements with various spacing etc. Import paths from
+ * valid statements are incrementing numbers, starting from 0,
+ * so that it is easy to verify validity of results.
+ */
+const sampleImportStatements = `
```

Comment:
New test cases can easily be added here. Currently it's assumed that the valid ones will use numeric characters for the import paths, so that the valid "paths" can be automatically pulled out and tested against.